medusa/restore_node.py

```diff
@@ -250,7 +251,7 @@ def table_is_allowed_to_restore(keyspace, table, fqtns_to_restore):
     return True
 
 
-def clean_path(p, use_sudo=True, keep_folder=False):
+def clean_path(p, use_sudo, keep_folder=False):
```

Comment:
`use_sudo` parameter is now mandatory to avoid errors detected during our tests when restoring with sstableloader: `use_sudo` was ignored due to https://github.com/teads/cassandra-medusa/blob/master/medusa/restore_node.py#L176

Reply:
Good call 👍